airbyte-cdk/bulk/toolkits/load-object-storage/src/main/kotlin/io/airbyte/cdk/load/command/object_storage/ObjectStorageFormatSpecification.kt

```diff
@@ -149,6 +150,7 @@ class CSVFormatSpecification(
 class AvroFormatSpecification(
     @JsonSchemaTitle("Format Type")
     @JsonProperty("format_type")
+    @JsonSchemaInject(json = """{"order":0}""")
```

Comment:
What is happening here? I guess `{"order":0}` gets merged into the generated schema, but what does it do?

Reply:
I assumed it fixes the order in which things are presented in the UI.

Reply:
I haven't actually verified that it's necessary or even honored tho. It could be left over from some hack.